aap_chatbot/src/AnsibleChatbot/AnsibleChatbot.tsx

```diff
@@ -319,6 +315,16 @@ export const AnsibleChatbot: React.FunctionComponent<ChatbotContext> = (
                   hasAttachButton={false}
                   hasStopButton={hasStopButton}
                   handleStopButton={handleStopButton}
+                  buttonProps={{
+                    //@ts-ignore
```

Comment:
These @ts-ignore tags are a workaround for https://github.com/patternfly/chatbot/issues/536